backend/data/processors/conditions_preprocessing.py

```diff
@@ -432,6 +442,134 @@ def map_word_to_program_type(processed, regex_word, type):
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only. Please contact the Co-op office for more information"
+
+    processed, number_of_subs = re.subn(
+    r"This course is by application only.", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only."
+
+    processed, number_of_subs = re.subn(
+    r"Its recommended to seek a progression check prior to application.", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "It is recommended to do a progression check prior to enrolling."
+
+    processed, number_of_subs = re.subn(
+    r"This course is by application only. Please visit Business School website for more information<br/><br/>", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only. Please visit Business School website for more information<br/><br/>"
+
+    processed, number_of_subs = re.subn(
+    r"This course is by application only.Please visit Business School website for more information<br/><br/>", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only. Please visit Business School website for more information<br/><br/>"
+
+
+    if re.search("good academic standing", processed, flags=re.IGNORECASE) != None:
+        note += "Students must be in good academic standing."
+    
+    processed = re.sub(
+    r"be in good academic standing", r"", processed, flags=re.IGNORECASE
+    )
+    
+    processed, number_of_subs = re.subn(
+    r"in their final year of a single or double Commerce degree", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "Students must be in their final year of a single or double Commerce degree"
+    processed, number_of_subs = re.subn(
+    r"Students are expected to be in their final year of a Bachelor of Commerce single or dual degree", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "Students must be in their final year of a single or double Commerce degree"
+    processed = re.sub(
+    r"Only available to students completing a Bachelor of Commerce as part of a single or double-degree", r"COMM#", processed, flags=re.IGNORECASE
+    )
+    processed = re.sub(
+    r"be enrolled in a Commerce Program", r"COMM#", processed, flags=re.IGNORECASE
+    )
+    
+    # need to change this/review these
+    processed = re.sub(
+    r"myBCom First Year Portfolio", r"", processed, flags=re.IGNORECASE
+    )
+    processed = re.sub(
+    r"with ", r"", processed, flags=re.IGNORECASE
+    )
+    processed, number_of_subs = re.subn(
+    r"completed 30 UOC of Integrated First Year Core<br/><br/>", r"", processed, flags=re.IGNORECASE
```

Comment:
This case is going to be handled by CIRCLES-262 so I'd recommend deleting this and just keeping the manual fix for COMM1999 around so that we don't have some redundant substitutions after 262 is finished